cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -646,6 +646,592 @@ func TestApplyTo(t *testing.T) {
 	}
 }
 
+func TestValidateControllersOptions(t *testing.T) {
+	testCases := []struct {
+		name         string
+		expectErrors bool
```

Comment:
I would suggest including a snippet of the expected error string related to the aspect being made invalid, and checking that `strings.Contains(err.Error(), expectErrorSubstring)`

Otherwise, the stub config getting validated might be getting rejected for an entirely unrelated reason and we'd not be testing what we expected